openupgrade_scripts/scripts/stock/16.0.1.1/post-migration.py

```diff
@@ -1,6 +1,27 @@
 from openupgradelib import openupgrade
 
 
+def _complete_stock_move_quantity_done_with_orm(env):
+    """In pre-migration we left out the moves with lines with different units of
+    measure to be treated by the ORM"""
+    env.cr.execute(
+        """
+        SELECT
+            move_id
+        FROM
+            stock_move_line
+        GROUP BY
+            move_id
+        HAVING
+            COUNT(DISTINCT product_uom_id) > 1 AND
+            SUM(qty_done) <> 0
```

Comment:
Please avoid this new SQL form that requires a lot of lines, and more for queries that are even logged (the same for the rest):

```suggestion
        SELECT move_id
        FROM stock_move_line
        GROUP BY move_id
        HAVING COUNT(DISTINCT product_uom_id) > 1
            AND SUM(qty_done) <> 0
```